tests/unit/test-class-wc-payments-order-service.php

```diff
@@ -37,16 +37,6 @@ public function set_up() {
 
 		$this->order_service = new WC_Payments_Order_Service( $this->createMock( WC_Payments_API_Client::class ) );
 		$this->order         = WC_Helper_Order::create_order();
-
-		// Turn on fraud settings.
-		update_option( 'wcpay_fraud_protection_settings_active', '1' );
```

Comment:
This flag was not checked anywhere in the codebase, just in tests. Likely a leftover from a cleanup task.